### PR TITLE
Feature/df 199 clean up liquibase scripts

### PR DESCRIPTION
**Pull request by birom**

### CODE REVIEW

File: alv-lamda-csv-importer/pom.xml [hunk 1]
```diff
@@ -112,9 +112,9 @@
             <plugin>
                 <groupId>org.liquibase</groupId>
                 <artifactId>liquibase-maven-plugin</artifactId>
-                <version>${liquibase.version}</version>
+                <version>3.5.1</version>
```
Comment (seco-mgabor) [inline alv-lamda-csv-importer/pom.xml]: Why didn't you change the property's value?

File: alv-lamda-csv-importer/src/main/resources/liquibase/changelog/20190617120000_initialize_schema.xml [hunk 2]
```diff
@@ -174,12 +192,12 @@
             <column name="num_vermittlungs_grad" type="numeric"/>
             <column name="qid_stes" type="numeric"/>
             <column name="q93_stes" type="VARCHAR(8)"/>
```
Comment (seco-mgabor) [inline alv-lamda-csv-importer/src/main/resources/liquibase/changelog/20190617120000_initialize_schema.xml]: What was wrong with boolean?

File: alv-lamda-csv-importer/src/main/java/ch/admin/seco/alv/basis/lamda/csv/importer/ImporterJobConfiguration.java [hunk 3]
```diff
@@ -118,6 +126,27 @@ private Step unzipStep() {
                 .build();
     }
```
Comment (seco-mgabor) [inline alv-lamda-csv-importer/src/main/java/ch/admin/seco/alv/basis/lamda/csv/importer/ImporterJobConfiguration.java]: I didn't find any test for this step.

**Comment by tonim42** [comment]
> 
> 
> I think an unit test would be good for the duplication lines.
> Weren't there some performance problems by this import? When yes, I'd expect an integration test to cover this situation.

Die Performance testen wir, indem wir den Batch laufen lassen, was mehrere Stunden dauert. Oderwie stellst Du Dir den Integrationstest sonst vor?

File: alv-lamda-csv-importer/pom.xml (hunk 1 above)
Comment (birom) [reply alv-lamda-csv-importer/pom.xml]: Fixed

File: alv-lamda-csv-importer/src/main/java/ch/admin/seco/alv/basis/lamda/csv/importer/ImporterJobConfiguration.java (hunk 3 above)
Comment (birom) [reply alv-lamda-csv-importer/src/main/java/ch/admin/seco/alv/basis/lamda/csv/importer/ImporterJobConfiguration.java]: fixed

File: alv-lamda-csv-importer/src/main/java/ch/admin/seco/alv/basis/lamda/csv/importer/ImporterJobConfiguration.java [hunk 4]
```diff
@@ -191,6 +216,27 @@ private ExecutionContext getJobExecutionContext(ChunkContext chunkContext) {
         return chunkContext.getStepContext().getStepExecution().getJobExecution().getExecutionContext();
     }
 
+    static void removeDuplicatedLinesFromFile(String parentDirectory, String fileName) {
```
Comment (seco-mgabor) [inline alv-lamda-csv-importer/src/main/java/ch/admin/seco/alv/basis/lamda/csv/importer/ImporterJobConfiguration.java]: Why static?

Comment (birom) [reply alv-lamda-csv-importer/src/main/java/ch/admin/seco/alv/basis/lamda/csv/importer/ImporterJobConfiguration.java]: Easier to test

**Pull request merged**